util/unifyfs/src/unifyfs-rm.c

```diff
@@ -567,6 +724,20 @@ static int invalid_terminate(unifyfs_resource_t* resource,
     return -ENOSYS;
 }
 
+/**
+ * @brief Default data stage routine
+ *
+ * @param resource  Not used
+ * @param args      Not used
+ *
+ * @return -ENOSYS
+ */
+static int invalid_stage(unifyfs_resource_t* resource,
+                         unifyfs_args_t* args)
+{
+    return -ENOSYS;
+}
+
```

Comment:
This function is unnecessary, and the return code is wrong for this context:
```
ENOSYS

“Function not implemented.” This indicates that the function called is not implemented at all,
either in the C library itself or in the operating system. When you get this error, you can be
sure that this particular function will always fail with ENOSYS unless you install a new version
of the C library or the operating system. 
```
Replace all callers with another error code.

Reply:
This mirrors the functions invalid_terminate, and invalid_stage that are in the dev branch.  If we need to change all of them let's open a new issue and create a new branch+PR.